test/e2e/rte/rte_local_test.go

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2022 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package rte
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/k8stopologyawareschedwg/resource-topology-exporter/pkg/nrtupdater"
+	"github.com/k8stopologyawareschedwg/resource-topology-exporter/pkg/resourcemonitor"
+	"github.com/k8stopologyawareschedwg/resource-topology-exporter/pkg/resourcetopologyexporter"
+)
+
+type ProgArgs struct {
+	NRTupdater      nrtupdater.Args
+	Resourcemonitor resourcemonitor.Args
+	RTE             resourcetopologyexporter.Args
+}
+
+var _ = Describe("[rte][local][config] RTE configuration", func() {
+	Context("with the binary available", func() {
+		It("should have any default for TM params", func() {
+			args := runConfig([]string{}, map[string]string{})
+			Expect(args.RTE.TopologyManagerScope).ToNot(BeEmpty())
+			Expect(args.RTE.TopologyManagerPolicy).ToNot(BeEmpty())
```

Comment:
moreover don't we want to make sure they have the default scope/policy (container/none)?

Reply:
that would make the test and the impl a bit more coupled, because both they will now need external knowledge of what's the correct default. Now only impl has this knowledge, the test does not and this makes it a bit weaker but also a bit more future proof. Do we want to check for the current defaults?

Reply:
right makes sense, thanks for clarifying!